src/gcam/components.py

```diff
@@ -1,39 +1,37 @@
-"""Definitions for the modules for GCAM and associated downstream models.
+"""Definitions for the components for GCAM and associated downstream models.
 
-We use the word "module" here in a bit of a nonstandard way.  A GCAM
-"module" is a functional unit that performs a specific step in the
-GCAM processing pipeline.  These modules are distinct from python
-modules; a GCAM module may be implemented in one or several python
-modules of its own (as in the water disaggregation module), or it may
-be completely contained in this module.
+A GCAM "component" is a functional unit that performs a specific step in
+the GCAM processing pipeline. A GCAM component may be implemented in one or
+several python components of its own (as in the water disaggregation
+component), or it may be completely contained in this component.
 
 Classes:
 
-GcamModuleBase    - Base class for all GCAM modules.  Provides the 
-                    interface, as well as services like managing
-                    threads, locks, and condition variables.
+GcamComponentBase     - Base class for all GCAM components.  Provides the
+                        interface, as well as services like managing
+                        threads, locks, and condition variables.
 
-GlobalParamsModule - Store parameters common to all modules.
+GlobalParamsComponent - Store parameters common to all components.
 
-GcamModule        - Run the GCAM core model.
+GcamComponent         - Run the GCAM core model.
 
-HydroModule       - Run the future hydrology calculation.
+HydroComponent        - Run the future hydrology calculation.
 
-HistoricalHydroModule - Run the historical hydrology calculation.
+HistoricalHydroComponent - Run the historical hydrology calculation.
 
-WaterDisaggregationModule - Run the water disaggregation calculation.
+WaterDisaggregationComponent - Run the water disaggregation calculation.
 
-NetcdfDemoModule  - Package outputs into a netCDF file for the 
-                    Decision Theater Demo.
+NetcdfDemoComponent   - Package outputs into a netCDF file for the
+                        Decision Theater Demo.
```

Comment:
Wow, the Decision Theater.  That brings back memories.  We can probably scrub out that mention too.